client/verta/verta/runtime.py

```diff
@@ -1,5 +1,7 @@
 # -*- coding: utf-8 -*-
-"""Classes and functions to enable logging within a model's predict() function.
+"""
+Classes and functions to enable logging within a model's predict() function.
+
```

Comment:
This blank line is needed for `.. versionadded` to render correctly.

Reply:
Ah, ok.   I see it.  I completely missed that.  Thanks!
![Screen Shot 2023-02-03 at 3 30 09 PM](https://user-images.githubusercontent.com/114943931/216723391-4220eea3-69b6-46e9-94c5-065f2b547699.png)